src/nft/components/explore/Table.tsx

```diff
@@ -50,12 +108,98 @@ export function Table<D extends Record<string, unknown>>({
   const navigate = useNavigate()
 
   useEffect(() => {
-    if (hiddenColumns && isMobile) {
-      setHiddenColumns(hiddenColumns)
+    if (!width) return
+
+    if (width < theme.breakpoint.sm) {
+      setHiddenColumns([
+        ColumnHeaders.Items,
+        ColumnHeaders.FloorChange,
+        ColumnHeaders.Volume,
+        ColumnHeaders.VolumeChange,
+        ColumnHeaders.Owners,
+      ])
+      setMaxCols(MAX_COLUMNS - 5)
```

Comment:
i would do this based on the length of the hidden columns. you can make that a variable

Reply:
so maybe in each of these you can just do `hiddenColumns = [...]`

then outside of the if statements you can do ```ts
setHiddenColumns(hiddenColumns)
setMaxCols(MAX_COLUMNS - hiddenColumns.length)
````

Reply:
but also i don't think you need maxCols? you can just get that information based on the hidden columns at render instead of needing new duplicated state for it

Reply:
you need maxCols because you aren't hiding all of the hiddenColumns at once - different breakpoints have a different amount of hidden columns